app/Http/routes.php

```
@@ -17,6 +17,18 @@
 use App\Slide;
 use App\Order;
 
+Route::group(['domain' => 'media.webumenia.{tld}'], function () {
+    Route::get('/', function ($tld) {
+        return "webumenia media server";
+    });
+    Route::get('{id}', function ($tld, $id) {
+        $item = Item::find($id);
+        if ($item) {
```

should we return a helpful error message if there is no item with that $id?